.github/workflows/php.yml

```
@@ -12,7 +12,7 @@ jobs:
     strategy:
       matrix:
         operating-system: ['ubuntu-latest', 'macos-latest']
-        php-versions: ['5.6', '7.0', '7.1', '7.2', '7.3', '7.4', '8.0', '8.1']
+        php-versions: ['8.1', '8.2']
```

```suggestion
        php-versions: ['8.1', '8.3']
```
by now it can be lower/upper set to 8.3